sdk/cosmos/azure-spring-data-cosmos-core/src/main/java/com/azure/spring/data/cosmos/repository/query/PartTreeCosmosQuery.java

```diff
@@ -40,9 +39,7 @@ protected CosmosQuery createQuery(CosmosParameterAccessor accessor) {
 
         final CosmosQuery query = creator.createQuery();
 
-        if (tree.isLimiting()) {
-            throw new NotImplementedException("Limiting is not supported.");
-        }
+        query.setTree(tree);
```

Comment:
We should pass the limit value to the query, instead of passing the PartTree object.